pkg/controller/storagecluster/reconcile.go

```diff
@@ -880,6 +880,10 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, nodeCount int, r
 			Placement: rook.PlacementSpec{
 				"all": defaults.DaemonPlacements["all"],
 				"mon": getCephDaemonPlacements(sc, "mon"),
+				"osd": getCephDaemonPlacements(sc, "osd"),
```

Comment:
We don't need the "osd" Placement, as that does not apply to PVC-based OSDs. Their Placement is entirely within the StorageClassDeviceSet of the CephCluster CR.

Reply:
Ah! Will fix that